cmd/kubeadm/app/kubeadm.go

```diff
@@ -30,6 +30,7 @@ func Run() error {
 	// We do not want these flags to show up in --help
 	pflag.CommandLine.MarkHidden("google-json-key")
 	pflag.CommandLine.MarkHidden("log-flush-frequency")
+	pflag.CommandLine.MarkHidden("fuzz-iters")
```

Comment:
if this is showing up, doesn't it mean kubeadm is linking to the api/testing package somehow? seems like the right fix would be to break that import chain